README.md

```diff
@@ -1,28 +1,26 @@
 [![Build status](https://ci.appveyor.com/api/projects/status/2fnekwfbm5h2ayk7/branch/master?svg=true)]
 
 # safe_examples
-Examples showcasing various features of the SAFE Network
+Examples showcasing various features of the SAFE Network.
 
 ## email_app (NodeJS - Electron)
-[email_app](email_app), written in `NodeJS`. Example application to exchange messages.
+[email_app](email_app), written in `NodeJS`, is an application to exchange messages.
 
 ## safe_web_api_playground (NodeJS - Electron)
-[safe_web_api_playground](safe_web_api_playground), written in `NodeJS`. Application to help in exploring the web api.
-
+[safe_web_api_playground](safe_web_api_playground), written in `NodeJS`, is an application allowing you to explore the web API.
 
 ## web_hosting_manager (NodeJS - Electron)
-[web_hosting_manager](web_hosting_manager), written in `NodeJS`. Example application to allow hosting and managing web files.
-
+[web_hosting_manager](web_hosting_manager), written in `NodeJS`, is an application which allows hosting and managing web files.
 
-For more details and build-instructions, please check the corresponding folder's README.
+**Note:** For more details and build-instructions, please check the corresponding folder's README.
 
 ## Packaging
 
-To run a complete package of the email/webhosting apps, (assuming each project has their own dependencies installed) you can run:
+To run a complete package of the email/webhosting apps, (assuming each project has their own dependencies installed) you can:
 
 * Run `yarn` in the root examples directory
 * Run `yarn install-all` to install and build `email_app` and `web_hosting_manager`.
 * Use either of the commands below to separately package each app:
 * Run `yarn pack:webhosting` to package and zip the app with all required files (to: web_hosting_manager/release).
 * Run `yarn pack:email` to package and zip the app with all required files (to: email_app/out).
-* Alternatively, run `yarn package-all` to package both `email_app` and `web_hosting_manager`
+* Alternatively, run `yarn package-all` to both of the `email_app` and `web_hosting_manager`packages.
```

Comment:
@username1001 Thank you Matt for submitting a PR to help us clarify readme.
This is the only line that I find confusing. It. makes me think that I need to run `yarn package-all` in each repository.

Reply:
Hey, @hunterlester, no worries, glad to help! :smile: I found that line confusing as well, so I changed it back to the original. Thanks!